feedsx/src/main/java/com/likeminds/feedsx/posttypes/util/PostTypeUtil.kt

```diff
@@ -631,23 +650,43 @@ object PostTypeUtil {
                 data
             )
 
+            val linkUri = Uri.parse(data.url) ?: return@apply
+            val linkText = linkUri.host?.lowercase(Locale.getDefault()) ?: data.url?.lowercase(
+                Locale.getDefault()
+            )
+
             val isImageValid = data.image.isImageValid()
             if (isImageValid) {
-                ivLink.show()
-                ImageBindingUtil.loadImage(
-                    ivLink,
-                    data.image,
-                    placeholder = R.drawable.ic_link_primary_40dp,
-                    cornerRadius = 8
-                )
+                try {
+                    ivLink.show()
+                    layoutNoImageFound.root.hide()
+                    ImageBindingUtil.loadImage(
+                        ivLink,
+                        data.image,
+                        placeholder = R.drawable.ic_link_primary_40dp,
+                        cornerRadius = 8
+                    ) {
```

Comment:
add a comment for this callback, that it is called when glide fails to render image